src/_includes/layouts/pattern-sidebar.hbs

```diff
@@ -0,0 +1,20 @@
+---
+title: Prototypes
+layout: layouts/pattern-library
+---
+
+<div class="EP_layout__sidebar">
+  <ul class="EP_sidebar__menu">
```

Comment:
I would expect there to be an `EP_sidebar` element for the BEM naming to be consistent. Maybe this needs to be broken into separate components more clearly?